meta-mender-toradex-nxp/README.md

```diff
@@ -0,0 +1 @@
+This is the Toradex machine layer for meta-mender-community.
```

Comment:
Please update this to the more common format in meta-mender-community.

Example: https://github.com/mendersoftware/meta-mender-community/tree/zeus/meta-mender-odroid